xdsl/interactive/app.py

```diff
@@ -161,6 +164,19 @@ def on_mount(self) -> None:
         # initialize gui to inital state - uncondensed mode.
         self.condense_mode = False
 
+        # initialize GUI with an interesting input IR and pass application
+        self.input_text_area.load_text(
+            """func.func @hello(%n : index) -> index {
```

Comment:
Can you please move this out to an UPPER_CAMEL_CASE constant at the level of the file? Maybe called `INITIAL_IR_TEXT`, or something like that?